client.go

```diff
@@ -403,6 +388,19 @@ func (c *client) periodicQuery(ctx context.Context, params *lookupParams) error
 			}
 			return ctx.Err()
 		}
+
+		if err := c.query(params); err != nil {
+			return err
+		}
+		// Exponential increase of the interval with jitter:
+		// the new interval will be between 1.5x and 2.5x the old interval, capped at maxInterval.
+		if interval != maxInterval {
```

Comment:
Note that backoff still jitters with randomness, even when it reached its max. I'm not worrying about it much though, just FYI.